### ISSUE TITLE
False positives and False negatives: Cannot add some language chains

### ISSUE THREAD

**Issue by nis-thac** [vitest-dev/eslint-plugin-vitest#675]
Related to #507 but not a duplicate, since I am using the latest version 1.1.36.
With all the examples, the expect gets marked with `ESLint: Expect has an unknown modifier (vitest/valid-expect)`.

My guess is that some or all of the [chai language chains](https://www.chaijs.com/api/bdd/#method_language-chains) are not treated properly.

```ts
// false positive
expect('hello').to.be.a('string').that.does.not.contain('world');
// false negative, vitest fails
// TypeError: expect(...).to.be is not a function
expect('hello').to.be('string').that.does.not.contain('world');


// false positive
expect('hello').not.to.be.an('array');


// true negative
expect('hello').to.not.contain('world');
// false positive
expect('hello').not.to.contain('world');
```

**Comment by RopoMen** [comment]
I have this same issue. Modifier chaining causes `vitest/valid-expect` error and removing modifier chaining removes the error.

**Comment by SampsonCrowley** [comment]
no answer?
